src/main.rs

```diff
@@ -1,3 +1,5 @@
+#![allow(warnings)]
```

Comment:
<img width="455" alt="image" src="https://github.com/user-attachments/assets/36bd2706-0f72-4a1b-90c0-5cbec222f211" />

Reply:
If there are a lot of warnings we could make a follow-up ticket to resolve them